src/routes/userRoutes.js

```diff
@@ -43,4 +45,13 @@ router
   .route('/:id')
   .delete(authenticateUser, authorizePermissions('admin'), deleteSingleUser);
 
+router.route('/:id/uploadUserImage').post(
```

Comment:
Since we're using `authenticateUser`, we are already requiring that the user is logged-in, and we can access that logged-in user's id with `req.user.userId`. 

I would suggest that it doesn't really make sense to also need the caller to pass in the user's id in the URL. It might also make it possible that a request passes in a different user's id in the URL than the requesting logged-in user. Maybe better to avoid the `:id` parameter altogether in this case

Reply:
@akosasante oh thank you , Akos!! got it! that absolutely makes sence